generator/test-attribute-errors/render_test.go

```diff
@@ -0,0 +1,57 @@
+package testattrerrs
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"errors"
+	"testing"
+
+	"github.com/a-h/templ"
+	"github.com/a-h/templ/generator/htmldiff"
+)
+
+//go:embed expected.html
+var expected string
+
+func Test(t *testing.T) {
+	t.Run("can render without error", func(t *testing.T) {
+		component := render(nil)
+
+		_, err := htmldiff.Diff(component, expected)
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+	})
+	t.Run("attribute expressions can return errors", func(t *testing.T) {
+		errSomethingBad := errors.New("bad error")
+
+		err := render(errSomethingBad).Render(context.Background(), &bytes.Buffer{})
```

Comment:
I think `render` is a bad name for a component. I think `component` would be better.

I probably used that name in the past, but now wish I hadn't. 😂

Reply:
Yep, I copied that from some other test, but I can change, no problem

Reply:
Changed to `TestComponent` as I saw in some other test files, let me know if you still you prefer `component`, I can change back